sdk/docs/manually-written/sdk/component-howtos/application-development/client-libraries/java-client-libraries.rst

```diff
@@ -54,4 +54,4 @@ Find the available versions of the Java client libraries in the Maven Central Re
 References
 -------------
 
-See the `JavaDoc reference documentation <javadocs/index.html>`_.
+See the `JavaDoc reference documentation </javadocs/3.3/index.html>`_.
```

Comment:
Let's clean these up in a later pass. Created issue here: https://github.com/DACH-NY/docs-website/issues/444